features/has_many.feature

```diff
@@ -26,6 +26,7 @@ Feature: Has Many
     When I go to the last author's show page
     And I follow "Edit User"
     Then I should see a link to "Add New Post"
+    And there should be 1 "input" tag within ".ui-sortable"
```

Comment:
I know this is a little awkward, but unfortunately I couldn't find any other observable change for when has many is initialized. The sortable option adds some classes that allowed me to make an assertion that fails before the changes in this PR and passes with the changes.

Reply:
It looks good enough to me! Easier that simulating a real drag and drop action.

Reply:
That makes sense. Will do

Reply:
Ok, how do you feel about this?